definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-solicitorAmendForRefusal-nonprod.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
Can you use the same name as mine?
AuthorisationCaseField-solicitorDn-nonprod.json (for example)
otherwise, we'll end up with multiple files for the same feature